core/arch/instr_shared.c

```diff
@@ -3487,7 +3487,7 @@ move_mm_avx512_reg_opcode(bool aligned64)
 #    ifdef X86
     /* move_mm_avx512_reg_opcode on unsupported processor. */
```

Comment:
I don't understand this comment: what is "unsupported processor": unsupported by DR??

Reply:
Changed the comment to "/* move_mm_avx512_reg_opcode can only be called on processors that support AVX-512. */"